sopel/plugin.py

```diff
@@ -524,7 +526,9 @@ def add_attribute(function):
     return add_attribute
 
 
-def echo(function=None):
+def echo(
+    function: typing.Optional[typing.Any] = None,
+) -> typing.Union[typing.Any, typing.Callable]:
```

Comment:
Why does the use of `typing.Any` bug me so much? I don't know, but it does.

I'll stop marking these now, in case there's a good explanation that will make me stop caring.